client/utils/apiWrapper.js

```diff
@@ -372,22 +372,22 @@ export const resendPIN = () => {
   }
 };
 
-export const userInfo = async () => {
-  const validUser = await checkValidUser();
```

Comment:
Please add the validUser check back

Reply:
valid user isn't being checked anywhere else in the function so i will just comment it out for now?